questions/templates/admin/login.html

```diff
@@ -0,0 +1,84 @@
+{% extends "admin/login.html" %}
+{% load i18n static %}
+
+{% load static %}
+
+{% block footer %}
+<style>
+    #footer {
+        text-align: center;
+        padding: 2rem;
+        background: #eee;
+        font-family: monospace;
+        font-weight: 400;
+        font-size: 1rem;
+        /* color: #333; */
+    }
+</style>
+<div id="footer">
+    Qujini Question Paper Generator
+</div>
+{% endblock %}
+{% block script %}
+
+
+<script src="{% static 'questions/js/login-extra.js' %}"></script>
+{% endblock %}
+
+
+<!-- //new start -->
+{% block branding %}
+<style>
+    body.login {
+        /* background: #ffc107; */
+        min-height: 100vh;
+        align-items: center;
+        justify-content: center;
+        display: flex;
+    }
+
+    .login #header {
+        height: auto;
+        padding: 1rem;
+        /* background: #222; */
+    }
+
+    .login #branding {
+        width: 100%;
+    }
+
+    .login .head {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+
+    .login #container {
+        /* background: #fff; */
+        border: none;
+        border-radius: 4px;
+        box-shadow: 2px 3px 5px #333;
+        overflow: hidden;
+        width: 50vw;
+        margin: 0;
+        max-width: 400px;
+    }
+
+    .login .submit-row {
+        display: flex;
+        justify-content: center;
+        padding: 2.5rem 0;
+    }
+
+    .login .submit-row input {
+        /* background: #ffc107; */
```

Comment:
Since this is a new css styling you were trying to implement, I suggest you should not leave unnecessary commented out attributes you might have used while working on it.

Reply:
@aswinshenoy from next time I will take care of that.